### ISSUE TITLE
Scrollbar moves Menu

### ISSUE THREAD

**Issue by GrimTheReaper** [excalidraw/excalidraw#240]
This might have been in another issue, and is more cosmetic, but whenever the menu expands horizontally and gains a scrollbar it puts the scrollbar on the inside of the menu which pushes the Shape menu closer together and is kind of jarring.

**Comment by dwelle** [comment]
Can you please show a video/screenshot or/and steps to reproduce?

**Comment by GrimTheReaper** [comment]
I would, but the scrollbar seems to no longer show for me?

**Comment by dwelle** [comment]
Alright. There might be a bug somewhere that'll pop up again, so feel free to comment here with further information and I'll reopen.

**Comment by dwelle** [comment]
Huh, so I spoke too soon as I can repro this myself:

1. open contextMenu so that it overflow beyond the window, which adds vertical scrollbar to window

Ideally, the contextMenu should (1) fit into the window (2) if it doesn't, have its own scrollbar.

I'll work on it myself later, if no one takes this.

**Comment by GrimTheReaper** [comment]
This is why I don't do frontend haha.

**Comment by dwelle** [comment]
And actually, that's probably different issue from the one you initially mentioned. The bug I've written up above is about viewport scrollbar.

The scrollbar on the sidePanel itself appears easily when there are too many items (e.g. when you select an element), and yes it bunches the shape icons together.

I'll probably drop the `align-items: space-between` rule now that the items are already overflowing by default, and revert to hardcoded margin.

**Comment by dwelle** [comment]
Closing as this is going to become obsolete after #343.
